modules/core/arrow-typeclasses/src/main/kotlin/arrow/typeclasses/Bifunctor.kt

```diff
@@ -23,3 +29,16 @@ private interface RightFunctor<F, X> : Functor<Kind<F, X>> {
   override fun <A, B> Kind2<F, X, A>.map(f: (A) -> B): Kind2<F, X, B> =
     F.run { bimap(::identity, f) }
 }
+
+private interface ComposedBifunctor<F, G, A, B> : Bifunctor<Kind2<F, Kind2<G, A, B>, Kind2<G, A, B>>> {
```

Comment:
This interface goes in `Composed.kt`, with the rest of the composed interfaces :)